src/example-types/facet.js

```diff
@@ -47,10 +47,15 @@ let setMatchOperators = (list, node) =>
   list.length > 1
     ? getMatchesForMultipleKeywords(list, node.optionsFilter)
     : {
-        [_.first(list)]: {
-          $regex: F.wordsToRegexp(node.optionsFilter),
-          $options: 'i',
-        },
+        $and: _.flow(
+          _.words,
+          _.map(option => ({
+            [_.first(list)]: {
+              $options: 'i',
+              $regex: option,
```

Comment:
Unless we wrap this with `.*` on both sides, I think we're changing the behavior - right now you can match mid-word (e.g. `Dade` matches `MiamiDade`)

Reply:
Tested locally and it works for a mid-word match

![Screen Shot 2021-05-06 at 10 39 26 PM](https://user-images.githubusercontent.com/532680/117390954-0cb25d80-aebd-11eb-9991-5f1b01c6170a.png)

Reply:
The regex is just the string so it should be fine:

![image](https://user-images.githubusercontent.com/910303/117393051-209f9480-aea8-11eb-9818-ccb14b22b5b6.png)

Reply:
Sweet! This is much simpler then 😁